ecosystem/platform/server/app/javascript/controllers/claim_nft_controller.ts

```diff
@@ -134,8 +134,9 @@ export default class extends Controller<HTMLAnchorElement> {
           const url = new URL(location.href);
           url.search = `connect-wallet`;
```

Comment:
I don't think I saw the PR that adds this - what's the point of adding ?connect-wallet to the URL? Seems like it should instead delete the `?wallet=` param from the URL (or just set `url.search = ''` if you don't want to construct the URLSearchParams object).

Reply:
Maybe I did it wrong and it somehow worked the way I wanted to. I just was trying to redirect back to the connect step.  Is setting `url.search = ''` better?

Reply:
Either way achieves the intended effect, the important thing is removing ?wallet= from the URL. I'm just saying there's nothing specifically looking for ?connect-wallet in the URL.